### ISSUE TITLE
Lagging in Android TV

### ISSUE THREAD

**Issue by sanjayajena37** [solid-software/flutter_vlc_player#447]
Hi All. Am facing lagging in android tv. pls help me. Am using 
  flutter_vlc_player: ^7.2.0 this one

**Comment by azulmouad** [comment]
same issues

**Comment by rawad1989** [comment]
Did you try to enable hardware acceleration with option `hwAcc: HwAcc.full`? for me when enabling it I get green screen when disabling it getting lagging in the video

**Comment by sanjayajena37** [comment]
Yes.

**Comment by azulmouad** [comment]
> Did you try to enable hardware acceleration with option `hwAcc: HwAcc.full`? for me when enabling it I get green screen when disabling it getting lagging in the video

Yes same issues,als the player is not working on 4K videos , Big screens TV
